apps/condo/domains/banking/schema/ImportBankTransactionsService.js

```diff
@@ -0,0 +1,201 @@
+        messageForUser: 'api.banking.importBankTransactions.BANK_ACCOUNT_HAS_NO_INTEGRATION',
+    },
+}
+
+const ImportBankTransactionsService = new GQLCustomSchema('ImportBankTransactionsService', {
+    doc: {
+        summary: 'Imports bank transactions from file in 1CClientBankExchange format of version 1.03',
+        errors,
+    },
+    types: [
+        {
+            access: true,
+            type: 'input ImportBankTransactionsInput { dv: Int!, sender: JSON!, file: Upload!, organizationId: ID! }',
+        },
+        {
+            access: true,
+            type: 'type ImportBankTransactionsOutput { bankAccount: BankAccount }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canImportBankTransactions,
+            schema: 'importBankTransactions(data: ImportBankTransactionsInput!): ImportBankTransactionsOutput',
+            resolver: async (parent, args, context) => {
+                const { data } = args
+                const { sender, file, organizationId } = data
+                const dvSender = { dv: 1, sender }
+                let conversionResult
+                try {
+                    const fileUpload = await Promise.resolve(file)
+                    const fileStream = fileUpload.createReadStream()
+                    conversionResult = await convertFrom1CExchangeToSchema(fileStream)
+                } catch (error) {
+                    throw new GQLError({
+                        ...errors.INVALID_FILE_FORMAT,
+                        messageInterpolation: {
+                            message: error.message,
+                        },
+                    }, context)
+                }
+                const { bankAccountData, bankTransactionsData } = conversionResult
+                const organization = await Organization.getOne(context, { id: organizationId })
+                const integration = await BankIntegration.getOne(context, { id: BANK_INTEGRATION_IDS['1CClientBankExchange'] })
+                let bankAccount = await BankAccount.getOne(context, {
+                    number: bankAccountData.number,
+                    organization: { id: organizationId },
+                })
+                let integrationContext
+                if (!bankAccount) {
+                    if (!integration) {
+                        throw new GQLError(errors.CANNOT_FIND_BANK_INTEGRATION, context)
+                    }
+                    integrationContext = await BankIntegrationContext.create(context, {
+                        ...dvSender,
+                        integration: { connect: { id: integration.id } },
+                        organization: { connect: { id: organizationId } },
+                    })
+                    bankAccount = await BankAccount.create(context, {
```

Comment:
Why there are no property connection? At this flow IMO we already know property. So it could be connected right there

Reply:
Yes, it can be, I will change it

Reply:
Added